content/docs/intro/languages/dotnet.md

```diff
@@ -0,0 +1,94 @@
+---
+title: ".NET"
+menu:
+  intro:
+    parent: languages
+    weight: 2
+
+aliases: ["/docs/reference/dotnet/"]
+---
+
+{{< langchoose dotnetonly >}}
+
+Pulumi supports programs written in any supported .NET language, including C#, F#, and Visual Basic. You can also use your favorite .NET tools such as editors, package managers, build systems, and test frameworks.
```

Comment:
Should we match the style of the existing Node.js and Python pages and be clear up front?

> Pulumi supports .NET programs running on .NET Core 3 and later.

Reply:
accepted